src/components/NcColorPicker/NcColorPicker.vue

```diff
@@ -176,6 +176,7 @@ const props = withDefaults(defineProps<{
 	 */
 	paletteOnly?: boolean
 }>(), {
+	advancedFields: true,
```

Comment:
boolean properties should not have `true` as the default.

Reply:
Yes